src/utils/layer-manager-utils.js

```diff
@@ -4,25 +4,31 @@ import { loadModules } from 'esri-loader';
 import { addLayerAnalyticsEvent, removeLayerAnalyticsEvent } from 'actions/google-analytics-actions';
 import { DEFAULT_OPACITY, LAYERS_CATEGORIES, layersConfig} from 'constants/mol-layers-configs';
 
-export const batchLayerManagerToggle = (layerNamesArray, activeLayers, callback, category) => {
-  const activeLayersNamesArray = activeLayers ? activeLayers.map(l => l.title) : [];
-  const areActive = activeLayers && intersection(layerNamesArray, activeLayersNamesArray).length > 0;
-  if (areActive) {
-    const updatedLayers = layerNamesArray.reduce((acc, title) => {
-      removeLayerAnalyticsEvent({ slug: title });
-      return [...acc.filter(l => l.title !== title)];
-    }, activeLayers);
-    callback({activeLayers: updatedLayers });
-  } else {
-    const layersToAdd = layerNamesArray.map(title => {
+// Toggles all the layers passed as ids on the first parameter
+export const batchLayerManagerToggle = (layerIdsToToggle, activeLayers, callback, category) => {
```

Comment:
I think we could add the styles in a different PR. This one is big enough

I will check this. So the previous batchToggle was either adding or removing layers. Not both. In this sense, it was not exactly being faithful to its name

Reply:
It was toggling the layers passed as param. They were being displayed or removed deoending on their previous state 🤷🏻‍♂️

Reply:
Maybe you're rigth and the name is missleading. The use case for this _batch toggle_ is for layers that need to be turned off or turned on all at once in the same action; this happens when a group of layers (i.e human pressures) are turned off from the legend. It also happens when community protected layers (that are two different layers) are turned on and off from the sidebar.
So yes it only turns on or off all the layers passed as param.
Maybe is better to call the function twice instead of complicating the logic inside of it?